docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -45,7 +45,10 @@ passing the data node id as parameter:
     # data_node == data_node_retrieved
     ```
 
-The data nodes that are part of a **scenario**, **pipeline** or **task** can be directly accessed as attributes:
+# Get data nodes by config_id
+
+The data nodes that are part of a **scenario**, **pipeline** or **task** can be directly accessed as attributes by
+using its config_id:
```

Comment:
"using its config_id" -> "using their configuration identifier:"

Reply:
I am not sure about replacing `config_id` with "configuration identifier" is good for the understanding. 
On the one hand, it s better English, I agree with that, but on the other hand, `config_id` is the name of the attribute. So it s easier to understand.

I would personally go for the name of the attribute name (with the appropriate styling like between \` or \_ ) for a better understanding.

Reply:
I agree with JR here, `config_id` to me is straight-forward that's (to my understanding) being used in multiple places within the doc :D